src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -290,3 +290,15 @@ function _formatLabel(meta: ProfileMeta): string {
   }
   return product + (version ? ` (${version})` : '') + (os ? ` ${os}` : '');
 }
+
+function _formatInterval(interval: number): string {
+  // Format in the most precise base (milliseconds, microseconds, or
+  // nanoseconds), to avoid cases where intervals are displayed with
+  // too many leading zeroes to be useful.
+  if (interval >= 1.0) {
+    return interval + 'ms';
+  } else if (interval * 1000 >= 1.0) {
+    return interval * 1000 + 'μs';
+  }
+  return interval * 1000000 + 'ns';
+}
```

Comment:
Thanks for this!

Can you please move it to `src/utils/format-numbers.js` and reuse existing `formatNano/Micro/Milliseconds` functions? This will make it easier to reuse and localize in the future.

You might need to adjust the parameters `significantDigits, maxFractionalDigits`.

Unfortunately we don't unit test this file yet, feel free to add a new test file for your new function if you want and to make it easier, but I won't make this a blocking comment.

Reply:
I moved it to `format-numbers.js`.  Not really sure on the new name, but it seems potentially reasonable.